tsconfig.json

```diff
@@ -1,26 +1,37 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "commonjs",
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
-    "paths": {
-      "include": [
-        "./types"
-      ]
-    },
-    "alwaysStrict": true,
-    "jsx": "react",
-    "allowJs": false,
+    /* Basic Options */
+    "target": "es2018",
+    "module": "esNext",
+    "lib": ["esnext", "dom"],
+    "allowJs": true,
```

Comment:
Do we actually need to `allowJs`?  We seem to be .ts/.tsx-only right now, and I'd be happy to keep it that way, unless there's something I'm missing.

Reply:
Yeah I think that's fine. This tsconfig comes originally from `meteor create --typescript` and I think I basically didn't edit it afterwards, but agree we shouldn't be using raw JS.